src/countdown/Countdown.tsx

```diff
@@ -0,0 +1,87 @@
+import "./_countdown.scss";
+
+import React, {useLayoutEffect} from "react";
+import classNames from "classnames";
+
+import useCountDownTimer from "../core/utils/hooks/useCountdownTimer";
+import List from "../list/List";
+import ListItem from "../list/item/ListItem";
+
+export interface CountdownProps {
+  testid: string;
+  startDate: Date;
+  countDownIntervalInSeconds?: number;
+  alwaysShowSeconds?: boolean;
+  onEnd?: () => void;
+  customClassName?: string;
+}
+
+interface CountDownItem {
+  id: "days" | "hours" | "minutes" | "seconds";
+  count: string | number;
+}
+
+function Countdown({
+  testid,
+  startDate,
+  countDownIntervalInSeconds = 1,
+  alwaysShowSeconds = false,
+  onEnd,
+  customClassName
+}: CountdownProps) {
+  const countDownData = useCountDownTimer(startDate, countDownIntervalInSeconds);
+  let items: CountDownItem[] = [];
+
+  if (countDownData.days >= 1) {
```

Comment:
Let's make it a utility to increase the readability

`const items = generateCountDownItems(countDownData);`